loctool.js

```diff
@@ -453,6 +462,16 @@ try {
 
         fs.writeFileSync(target.getPath(), target.serialize(), "utf-8");
         break;
+
+    case "generate":
+        var genResource = new GenerateResource({
+            targetDir: settings.targetDir,
+            xliffsDir: settings.xliffsDir,
+            resRoot: settings.xliffResRoot,
+            resName: settings.xliffResName
+        });
+        genResource.init();
```

Comment:
`init()` and `generate()` should be called separately so that they can be unit tested independently